sdk/appcenter-codepush/src/androidTest/java/com/microsoft/appcenter/codepush/CodePushTest.java

```diff
@@ -0,0 +1,17 @@
+package com.microsoft.appcenter.codepush;
+
+import android.support.test.runner.AndroidJUnit4;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static org.junit.Assert.*;
```

Comment:
never use star imports, please configure your commit dialog to cleanup imports and format code etc... or do it manually with IDE actions

Reply:
Fixed it, thanks.